Tasks/AzurePowerShellV4/package.json

```diff
@@ -8,8 +8,8 @@
         "@types/node": "^10.17.0",
         "@types/q": "1.0.7",
         "azure-pipelines-task-lib": "^3.0.6-preview.0",
-        "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.0-preview.0",
-        "azure-pipelines-tasks-utility-common": "^3.0.0-preview.0"
+        "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.4",
+        "azure-pipelines-tasks-utility-common": "^3.0.3"
```

Comment:
You can remove this package from dependency as well.

Reply:
As suggested made the changes.